wgpu-native/src/command/mod.rs

```diff
@@ -99,8 +99,8 @@ impl CommandBufferHandle {
         base: &mut TrackerSet,
         head: &TrackerSet,
         stitch: Stitch,
-        buffer_guard: &Storage<BufferHandle>,
-        texture_guard: &Storage<TextureHandle>,
+        buffer_guard: &Storage<BufferHandle, crate::BufferId>,
```

Comment:
let's be consistent with use of other *Id types and just `use` it at the top instead of having absolute paths in the code